src/components/common/IntegrationCard/IntegrationCard.tsx

```diff
@@ -0,0 +1,57 @@
+"use client";
+
+import Image from "next/image";
+import { useState } from "react";
+
+type CardProperties = {
+  heading: string;
+  description: string;
+  logoURL: string;
+  isActive: boolean;
+};
+export const TOGGLE_BTN_TEST_ID: string = "toggleBtn";
+export const INTEGRATION_CARD_TEST_ID: string = "integrationCard";
+export const INTEGRATION_CARD_IMAGE_TEST_ID: string = "integrationCardImage";
+export const IntegrationCard = (properties: CardProperties) => {
+  const [isActive, setisActive] = useState<boolean>(properties.isActive);
+  const handleToogle = () => {
+    if (isActive) {
+      setisActive(false);
+    } else {
+      setisActive(true);
+    }
+  };
+
+  return (
+    <div
+      data-testid={INTEGRATION_CARD_TEST_ID}
+      className="flex h-[159.29px] w-[341px] flex-col gap-6 rounded-lg border-[1px] border-border px-4 py-6 text-foreground"
```

Comment:
This component wouldn't be responsive. Use full width, then you can set the max width as 341px

Reply:
Done